client/box/box.ctrl.js

```diff
@@ -79,6 +84,15 @@ module.exports = class Box {
       .tap(() => this.onscroll())
       .then(() => this.$scope.$apply());
   }
+  fetchIcons () {
+    return this.io.socket.getAsync(
+      `/api/v1/box/${this.id}?pokemonFields=${POKEMON_FIELDS_FOR_CARD}`
+    ).then(data => {
+      Object.assign(this.data, data);
+    }).catch(err => {
+      this.errorStatusCode = err.statusCode;
```

Comment:
Does `this.errorStatusCode` actually do anything on the page with the list of boxes? It's used to display a 404 when viewing a particular box, but it's unclear what it's doing in this context.